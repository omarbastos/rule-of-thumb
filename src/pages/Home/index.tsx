import HeaderHero from '../../layouts/HeroHeader/HeroHeader'
import BannerTop from '../../components/BannerTop/index'
import BannerBottom from '../../components/BannerBottom/index'

import { Poll } from 'utils/interfaces'
import useGetPolls from 'utils/hooks/useGetPolls'
import CardSquare from 'components/base/Cards/Square'
import GridView from 'components/GridView'
import DropdownMenu from 'components/DropdownMenu'
import { VIEWS } from '../../utils/constants'
import { useEffect, useState } from 'react'
import stg from 'utils/strings'
import useWindowDimensions from '../../utils/hooks/useWindowDimensions'
import ListView from 'components/ListView'

interface Props {}

const Home = (props: Props) => {
  const { polls } = useGetPolls()
  const [selectedView, setSelectedView] = useState(VIEWS.LIST)
  const { isMobile } = useWindowDimensions()
  useEffect(() => {
    if (isMobile) {
      setSelectedView(VIEWS.GRID)
    }

    return () => setSelectedView(VIEWS.LIST)
  }, [isMobile])

  return (
    <>
      <HeaderHero />
      <div className="max-centered">
        <BannerTop />
        <main role="main">
          <div className="flex justify-between my-4 md:my-8">
            <h3 className="flex-auto text-2xl md:text-4xl font-light text-[#464646]">
              {stg('previous_rulings')}
            </h3>
            {!isMobile && (
              <DropdownMenu
                selected={selectedView}
                setSelected={setSelectedView}
              />
            )}
          </div>
          {selectedView === VIEWS.LIST ? (
            <ListView polls={polls} />
          ) : (
            <GridView polls={polls} />
          )}
        </main>
        <BannerBottom />
      </div>
    </>
  )
}

export default Home
