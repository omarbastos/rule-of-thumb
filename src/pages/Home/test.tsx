import { render, screen } from '@testing-library/react'

import Home from '.'

describe('<Home />', () => {
  it('should render Home as default', () => {
    const { container } = render(<Home />)

    expect(container).toMatchSnapshot()
  })
  it('should render the Home as mobile', () => {
    const { container } = render(<Home />)
    const resizeWindow = (width: number, height: number) => {
      window = Object.assign(window, { innerWidth: width })
      window = Object.assign(window, { innerHeight: height })

      window.dispatchEvent(new Event('resize'))
    }
    resizeWindow(375, 600)
    expect(container).toMatchSnapshot()
  })
})
