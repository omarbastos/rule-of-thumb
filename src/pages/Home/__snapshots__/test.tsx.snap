// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should render Home as default 1`] = `
<div>
  <header
    class="hero"
  >
    <img
      alt="Pope Francis"
      class="hero__background max-w-fit md:max-w-full"
      sizes="(min-width: 750px) 1440px, 100vw"
      src="assets/img/pope-francis.png"
      srcset=" 750w,  1440w"
    />
    <div
      class="max-centered"
    >
      <div
        class="hero__featured-card"
      >
        <div
          class="featured-card__glass-background"
        />
        <div
          class="featured-card__content"
        >
          <p
            class="featured-card__hairline"
          >
            What's your opinion on
          </p>
          <h2
            class="featured-card__title"
          >
            Pope Francis
            ?
          </h2>
          <p
            class="featured-card__desc"
          >
            He’s talking tough on clergy sexual abuse, or is he just another
  pervert protector? (thumbs down) or a true pedophile punishing
  pontiff? (thumbs up)
          </p>
          <p
            class="featured-card__more-info"
          >
            <a
              class="flex underline"
              href="http://wikipedia.com"
            >
              <svg
                class="featured-card__more-info-icon"
                viewBox="0 0 27 18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M27 .303c0 .1-.032.2-.09.28a.255.255 0 01-.2.125 2.46 2.46 0 00-1.453.602 5.676 5.676 0 00-1.166 1.952l-6.127 14.533c-.04.135-.152.203-.337.203a.374.374 0 01-.337-.203l-3.436-7.564-3.952 7.564a.374.374 0 01-.337.203.34.34 0 01-.349-.203L3.196 3.262a5.052 5.052 0 00-1.19-1.89A3.161 3.161 0 00.267.708.23.23 0 01.086.6.378.378 0 010 .355C0 .118.064 0 .192 0 .73 0 1.29.025 1.876.075c.544.053 1.056.078 1.536.078.49 0 1.067-.026 1.732-.078C5.839.025 6.456 0 6.994 0c.128 0 .192.118.192.355 0 .235-.04.352-.119.352a2.308 2.308 0 00-1.268.43c-.297.22-.47.581-.463.963.015.263.08.521.192.757l4.975 11.826 2.824-5.614-2.631-5.807A7.637 7.637 0 009.53 1.257a2.274 2.274 0 00-1.382-.55A.208.208 0 017.986.6a.4.4 0 01-.078-.245c0-.237.054-.355.168-.355.494-.002.987.023 1.477.075.46.054.92.08 1.382.078.48 0 .988-.026 1.525-.078C13.013.025 13.558 0 14.094 0c.128 0 .192.118.192.355 0 .235-.038.352-.119.352-1.073.078-1.61.399-1.61.963.047.414.174.814.373 1.175l1.74 3.72 1.732-3.403c.209-.37.333-.786.36-1.215 0-.775-.536-1.188-1.61-1.24-.097 0-.144-.117-.144-.352a.44.44 0 01.071-.24c.05-.077.098-.115.145-.115.385 0 .857.025 1.418.075.536.053.978.078 1.322.078a13.6 13.6 0 001.093-.065A16.806 16.806 0 0120.584 0c.095 0 .142.1.142.303 0 .27-.088.405-.263.405-.54.036-1.061.224-1.508.544a6.937 6.937 0 00-1.423 2.01l-2.308 4.492 3.125 6.702 4.614-11.294c.149-.36.23-.745.24-1.137 0-.828-.537-1.267-1.61-1.317-.097 0-.145-.118-.145-.353 0-.237.071-.355.216-.355.392 0 .857.025 1.394.075.496.053.914.078 1.25.078.409-.003.818-.03 1.224-.078.483-.05.915-.075 1.3-.075.111 0 .168.1.168.303z"
                  fill="#FFF"
                  fill-rule="nonzero"
                />
              </svg>
              More information
            </a>
          </p>
          <p
            class="featured-card__cta"
          >
            What’s Your Veredict?
          </p>
          <div
            class="featured-card__buttons"
          >
            <button
              aria-label="thumbs up"
              class="icon-button"
            >
              <img
                alt="thumbs up"
                src=""
              />
            </button>
            <button
              aria-label="thumbs down"
              class="icon-button"
            >
              <img
                alt="thumbs down"
                src=""
              />
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="hero__closing-gauge"
    >
      <div
        class="closing-gauge__left"
      >
        <span
          class="closing-gauge__title"
        >
          closing in
        </span>
      </div>
      <div
        class="closing-gauge__right"
      >
        <span
          class="closing-gauge__number"
        >
          22
        </span>
        <span
          class="closing-gauge__desc"
        >
          days
        </span>
      </div>
    </div>
  </header>
  <div
    class="max-centered"
  >
    <aside
      aria-label="Speak Out"
      class="banner banner-top"
      role="doc-tip"
    >
      <div
        class="banner__left"
      >
        <span
          class="block text-sm whitespace-nowrap font-light"
        >
          Speak out. Be heard.
        </span>
        <span
          class="block text-2xl whitespace-nowrap font-bold"
        >
          Be counted
        </span>
      </div>
      <div
        class="banner__right"
      >
        <p
          class="text-xs md:text-base text-[#464646] font-light"
        >
          Rule of Thumb is a crowd sourced court of public opinion where anyone
  and everyone can speak out and speak freely. It’s easy: You share your
  opinion, we analyze and put the data in a public report.
        </p>
      </div>
      <button
        aria-label="close"
        class="icon-button"
      >
        <svg
          height="20"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="none"
            fill-rule="evenodd"
            stroke="#000"
            stroke-width="2"
          >
            <path
              d="M1 19L19 1M1 1l18 18"
            />
          </g>
        </svg>
      </button>
    </aside>
    <main
      role="main"
    >
      <div
        class="flex justify-between my-4 md:my-8"
      >
        <h3
          class="flex-auto text-2xl md:text-4xl font-light text-[#464646]"
        >
          Previous Rulings
        </h3>
        <div
          class="relative flex-none"
        >
          <button
            aria-label="View"
            class="w-40 bg-white text-sm py-2 px-4 whitespace-nowrap text-black border-solid border-2 border-black font-normal flex justify-center items-center"
            data-testid="button-dropdow-menu"
            type="button"
          >
            <span
              class="flex-auto"
            >
               
              List
            </span>
            <svg
              class=" flex-none justify-self-end"
              fill="none"
              height="7"
              viewBox="0 0 11 7"
              width="11"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M5.25 7L6.11959e-07 -9.17939e-07L10.5 0L5.25 7Z"
                fill="#333333"
                fill-rule="evenodd"
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        class="grid grid-cols-1"
      >
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
      </div>
    </main>
    <aside
      aria-label="Submit a name"
      class="banner banner-bottom"
      role="doc-tip"
    >
      <img
        alt="people"
        class="banner__background"
        role="none"
        sizes="(min-width: 750px) 1440px, 100vw"
        src=""
        srcset=" 750w,   1440w"
      />
      <div
        class="banner__left"
      >
        <h2
          class="text-lg text-center font-normal"
        >
           Is there anyone else you would want us to add?
        </h2>
      </div>
      <div
        class="banner__right"
      >
        <button
          aria-label="Submit a name"
          class="text-base text-center border border-b-2 border-black p-2 my-2 w-full"
        >
          Submit a name
        </button>
      </div>
    </aside>
  </div>
</div>
`;

exports[`<Home /> should render the Home as mobile 1`] = `
<div>
  <header
    class="hero"
  >
    <img
      alt="Pope Francis"
      class="hero__background max-w-fit md:max-w-full"
      sizes="(min-width: 750px) 1440px, 100vw"
      src="assets/img/pope-francis.png"
      srcset=" 750w,  1440w"
    />
    <div
      class="max-centered"
    >
      <div
        class="hero__featured-card"
      >
        <div
          class="featured-card__glass-background"
        />
        <div
          class="featured-card__content"
        >
          <p
            class="featured-card__hairline"
          >
            What's your opinion on
          </p>
          <h2
            class="featured-card__title"
          >
            Pope Francis
            ?
          </h2>
          <p
            class="featured-card__desc"
          >
            He’s talking tough on clergy sexual abuse, or is he just another
  pervert protector? (thumbs down) or a true pedophile punishing
  pontiff? (thumbs up)
          </p>
          <p
            class="featured-card__more-info"
          >
            <a
              class="flex underline"
              href="http://wikipedia.com"
            >
              <svg
                class="featured-card__more-info-icon"
                viewBox="0 0 27 18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M27 .303c0 .1-.032.2-.09.28a.255.255 0 01-.2.125 2.46 2.46 0 00-1.453.602 5.676 5.676 0 00-1.166 1.952l-6.127 14.533c-.04.135-.152.203-.337.203a.374.374 0 01-.337-.203l-3.436-7.564-3.952 7.564a.374.374 0 01-.337.203.34.34 0 01-.349-.203L3.196 3.262a5.052 5.052 0 00-1.19-1.89A3.161 3.161 0 00.267.708.23.23 0 01.086.6.378.378 0 010 .355C0 .118.064 0 .192 0 .73 0 1.29.025 1.876.075c.544.053 1.056.078 1.536.078.49 0 1.067-.026 1.732-.078C5.839.025 6.456 0 6.994 0c.128 0 .192.118.192.355 0 .235-.04.352-.119.352a2.308 2.308 0 00-1.268.43c-.297.22-.47.581-.463.963.015.263.08.521.192.757l4.975 11.826 2.824-5.614-2.631-5.807A7.637 7.637 0 009.53 1.257a2.274 2.274 0 00-1.382-.55A.208.208 0 017.986.6a.4.4 0 01-.078-.245c0-.237.054-.355.168-.355.494-.002.987.023 1.477.075.46.054.92.08 1.382.078.48 0 .988-.026 1.525-.078C13.013.025 13.558 0 14.094 0c.128 0 .192.118.192.355 0 .235-.038.352-.119.352-1.073.078-1.61.399-1.61.963.047.414.174.814.373 1.175l1.74 3.72 1.732-3.403c.209-.37.333-.786.36-1.215 0-.775-.536-1.188-1.61-1.24-.097 0-.144-.117-.144-.352a.44.44 0 01.071-.24c.05-.077.098-.115.145-.115.385 0 .857.025 1.418.075.536.053.978.078 1.322.078a13.6 13.6 0 001.093-.065A16.806 16.806 0 0120.584 0c.095 0 .142.1.142.303 0 .27-.088.405-.263.405-.54.036-1.061.224-1.508.544a6.937 6.937 0 00-1.423 2.01l-2.308 4.492 3.125 6.702 4.614-11.294c.149-.36.23-.745.24-1.137 0-.828-.537-1.267-1.61-1.317-.097 0-.145-.118-.145-.353 0-.237.071-.355.216-.355.392 0 .857.025 1.394.075.496.053.914.078 1.25.078.409-.003.818-.03 1.224-.078.483-.05.915-.075 1.3-.075.111 0 .168.1.168.303z"
                  fill="#FFF"
                  fill-rule="nonzero"
                />
              </svg>
              More information
            </a>
          </p>
          <p
            class="featured-card__cta"
          >
            What’s Your Veredict?
          </p>
          <div
            class="featured-card__buttons"
          >
            <button
              aria-label="thumbs up"
              class="icon-button"
            >
              <img
                alt="thumbs up"
                src=""
              />
            </button>
            <button
              aria-label="thumbs down"
              class="icon-button"
            >
              <img
                alt="thumbs down"
                src=""
              />
            </button>
          </div>
        </div>
      </div>
    </div>
    <div
      class="hero__closing-gauge"
    >
      <div
        class="closing-gauge__left"
      >
        <span
          class="closing-gauge__title"
        >
          closing in
        </span>
      </div>
      <div
        class="closing-gauge__right"
      >
        <span
          class="closing-gauge__number"
        >
          22
        </span>
        <span
          class="closing-gauge__desc"
        >
          days
        </span>
      </div>
    </div>
  </header>
  <div
    class="max-centered"
  >
    <aside
      aria-label="Speak Out"
      class="banner banner-top"
      role="doc-tip"
    >
      <div
        class="banner__left"
      >
        <span
          class="block text-sm whitespace-nowrap font-light"
        >
          Speak out. Be heard.
        </span>
        <span
          class="block text-2xl whitespace-nowrap font-bold"
        >
          Be counted
        </span>
      </div>
      <div
        class="banner__right"
      >
        <p
          class="text-xs md:text-base text-[#464646] font-light"
        >
          Rule of Thumb is a crowd sourced court of public opinion where anyone
  and everyone can speak out and speak freely. It’s easy: You share your
  opinion, we analyze and put the data in a public report.
        </p>
      </div>
      <button
        aria-label="close"
        class="icon-button"
      >
        <svg
          height="20"
          width="20"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="none"
            fill-rule="evenodd"
            stroke="#000"
            stroke-width="2"
          >
            <path
              d="M1 19L19 1M1 1l18 18"
            />
          </g>
        </svg>
      </button>
    </aside>
    <main
      role="main"
    >
      <div
        class="flex justify-between my-4 md:my-8"
      >
        <h3
          class="flex-auto text-2xl md:text-4xl font-light text-[#464646]"
        >
          Previous Rulings
        </h3>
      </div>
      <div
        class="grid grid-cols-1"
      >
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
        <div
          class="w-60 h-24 border-2 rounded-md mx-auto my-2"
        >
          <div
            class="flex animate-pulse flex-row items-center h-full justify-center space-x-5"
          >
            <div
              class="w-12 bg-gray-300 h-12 rounded-full "
            />
            <div
              class="flex flex-col space-y-3"
            >
              <div
                class="w-36 bg-gray-300 h-6 rounded-md "
              />
              <div
                class="w-24 bg-gray-300 h-6 rounded-md "
              />
            </div>
          </div>
        </div>
      </div>
    </main>
    <aside
      aria-label="Submit a name"
      class="banner banner-bottom"
      role="doc-tip"
    >
      <img
        alt="people"
        class="banner__background"
        role="none"
        sizes="(min-width: 750px) 1440px, 100vw"
        src=""
        srcset=" 750w,   1440w"
      />
      <div
        class="banner__left"
      >
        <h2
          class="text-lg text-center font-normal"
        >
           Is there anyone else you would want us to add?
        </h2>
      </div>
      <div
        class="banner__right"
      >
        <button
          aria-label="Submit a name"
          class="text-base text-center border border-b-2 border-black p-2 my-2 w-full"
        >
          Submit a name
        </button>
      </div>
    </aside>
  </div>
</div>
`;
